R/recode_into.r

```diff
@@ -0,0 +1,109 @@
+#' @title Recode values from one or more variables into a new variable
+#' @name recode_into
+#'
+#' @description
+#' This functions recodes values from one or more variables into a new variable.
+#' It is a convenient function to avoid nested [`ifelse()`] statements.
+#'
+#' @param ... A sequence of two-sided formulas, where the left hand side (LHS)
+#' is a logical condition that determines which values match this case. The
+#' right hand side (RHS) indicates the replacement value. The last formula can
+#' be used to indicate the default value. If the `default` argument is provided,
+#' the last formula is considered as further recode-pattern.
+#' @param data Optional, name of a data frame. This can be used to avoid writing
+#' the data name multiple times in `...`. See 'Examples'.
+#' @param default Indicates the default value that is chosen when no match in
+#' the formulas in `...` is found. If `NULL`, the default value needs to be
+#' determined as last formula in `...`. See 'Examples'.
+#'
+#' @return A vector with recoded values.
+#'
+#' @examples
+#' x <<- 1:30
```

Comment:
Should be `<-`?

Reply:
No, else examples are failing. It's again one of these environment issues.

Reply:
Ah ok